core/src/main/scala/org/broadinstitute/dsde/rawls/model/SpendReportingModel.scala

```diff
@@ -0,0 +1,36 @@
+package org.broadinstitute.dsde.rawls.model
+
+import org.broadinstitute.dsde.workbench.model.google.{BigQueryDatasetName, GoogleProject}
+import org.broadinstitute.dsde.workbench.model.google.GoogleModelJsonSupport._
+import org.joda.time.DateTime
+import spray.json.DefaultJsonProtocol._
+
+case class BillingProjectSpendConfiguration(datasetGoogleProject: GoogleProject, datasetName: BigQueryDatasetName)
+
+case class BillingProjectSpendExport(billingProjectName: RawlsBillingProjectName, billingAccountId: RawlsBillingAccountName, spendExportTable: Option[String])
+
+case class SpendReportingResults(spendDetails: Seq[SpendReportingAggregation], spendSummary: SpendReportingForDateRange)
+case class SpendReportingAggregation(aggregationKey: SpendReportingAggregationKey, spendData: Seq[SpendReportingForDateRange])
+case class SpendReportingForDateRange(
+                                       cost: String,
+                                       credits: String,
+                                       currency: String,
+                                       startTime: DateTime,
+                                       endTime: DateTime
+                                     )
+
+case class SpendReportingAggregationKey(key: String)
```

Comment:
Can you add some documentation about what this is intended to be?

Reply:
Could we have an enumeration of the supported aggregation keys?

Reply:
Yeah, I've been thinking that it might be a good idea to make this an enum. I was going to wait until I add per-workspace reporting when we have multiple aggregation keys

Reply:
It's fine with me if you wait until then.